### DISCUSSION TITLE
Unable to Connect to Audible

### DISCUSSION THREAD

**Discussion by gottifire** [openaudible/openaudible#949] [General]
Just updated to the latest beta version

When I click "connect to audible" a box comes up asking me to choose my region. when I click "ok" it looks like a download progression box comes up for a split second then disappears. I just renewed my license for openaudible and closed the program then reopened but still unable to connect

**Comment by openaudible** [comment]
Look at the console and see if any errors are reported. You can search for key words In the error in our “issues” to see if there is a solution. 
Are you on Linux? 

OA uses an internal web browser to log you in and the browser widget has several variants and compatibility issues with some systems. 

Sometimes you can install an update and it will start working, or there are other work arounds. 

We don’t look at the discussion as much as issues, so apologies for the late reply.
